### PR TITLE
Test ArborX install as part of the CI

### LINKED ISSUE #12: Test ArborX installation

### PR DESCRIPTION
**Pull request by dalg24**
Fix #12

### CODE REVIEW

File: .jenkins [hunk 1]
```diff
@@ -82,6 +80,10 @@ pipeline {
                             sh 'ccache --show-stats'
                             xunit([CTest(deleteOutputFiles: true, failIfNotNew: true, pattern: 'build/Testing/**/Test.xml', skipNoTestFiles: false, stopProcessingIfError: true)])
                         }
+                        success {
+                            sh 'cd build && make install'
+                            sh 'rm -rf test_install && mkdir -p test_install && cd test_install && cmake -D CMAKE_CXX_COMPILER_LAUNCHER=ccache -D CMAKE_CXX_COMPILER=clang++ -D CMAKE_PREFIX_PATH="$KOKKOS_DIR;$ARBORX_DIR" ../examples && make VERBOSE=1 && make test'
```
Comment (aprokop) [inline .jenkins]: `make -j8`? and what is `make test`?

Comment (dalg24) [reply .jenkins]: > `make -j8`?

There is only one cpp file at the moment.

> what is `make test`?

CMake create this rule by default and it calls CTest.

**Comment by dalg24** [comment]
> I would go further and physically copy the examples/ directory out of the ArborX tree so that if we do something like #include <../benchmarks/point_clouds/point_clouds.hpp> by accident, that will no longer work.

Overkill.

**Comment by aprokop** [comment]
> Overkill.

Disagree. And easy to implement.

**Pull request merged**